### PR TITLE
Broken Link issue fixed

### LINKED ISSUE #803: Broken Link Need to fix
## Bug Report
*Problem*
[This page ](https://github.com/kubernetes-sigs/krew/blob/master/CONTRIBUTING.md) has one broken link of `contributor cheat sheet` which needs to fix.
 

*solution*
need to locate the correct path of README.

/assign 
/kind bug
/kind documentation

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>

Fixes #803

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
 - [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
-- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet.md) - Common resources for existing developers
+- [Contributor Cheat Sheet](https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README.md) - Common resources for existing developers
```
Comment (ahmetb) [inline CONTRIBUTING.md:17]: Please keep git.k8s.io

Comment (amolmote) [reply CONTRIBUTING.md:17]: sure thank you for suggestion

Comment (amolmote) [reply CONTRIBUTING.md]: done @ahmetb

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**